packages/components/src/toggle-group-control/toggle-group-control-option-base/styles.ts

```diff
@@ -70,7 +70,7 @@ export const buttonView = ( {
 	}
 
 	&:active {
-		background: ${ CONFIG.controlBackgroundColor };
```

Comment:
Removing this `CONFIG` variable altogether since this is the only place it's used. (Part of #43997)